tests/test_global_opts.rs

```diff
@@ -235,5 +235,6 @@ fn test_help() {
           --at-operation <AT_OPERATION>  Operation to load the repo at [default: @] [aliases: at-op]
           --color <WHEN>                 When to colorize output (always, never, auto)
           --config-toml <TOML>           Additional configuration options
+      -v, --verbose                      Enable verbose logging
```

Comment:
Can you add some test specifically for verbose logging? Just making sure that we actually get verbose logging when we ask for it. Maybe also with the EnvFilter?

Reply:
`test_verbose_logging_enabled` added at the end of the `test_global_opts` file. Turning on verbose logging now also logs out a "verbose logging enabled" message in order to properly test this (and it might give users more comfort if they're not seeing any other verbose logs).